extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -114,6 +131,46 @@ export function VideoWrapper({
     };
   }, [mediasession, playing, metadata, play, pause]);
 
+  useImperativeHandle(
+    ref,
+    () =>
+      Object.defineProperties(
```

Comment:
You don't need `defineProperties`. You can just do:

```
{
  play,
  pause,
  ...,
  get currentTime() {
    return ...
  },
  ...
}
```

Reply:
🤯